src/App.php

```diff
@@ -203,7 +212,8 @@ protected function configureIoc( \Slim\Helper\Set $container ) {
 	protected function configureView( View $view ) {
 		$view->replace( [
 			'app' => $this->slim,
-			'i18nCtx' => $this->slim->i18nContext
+			'i18nCtx' => $this->slim->i18nContext,
+			'supportedLanguages' => $this->getSupportedLanguages(),
```

Comment:
does the trailing comma violate any coding conventions (not sure)? I know some organizations prefer it this way